libs/framework/src/bundle_archive.c

```diff
@@ -65,6 +66,7 @@ static celix_status_t celix_bundleArchive_storeBundleStateProperties(bundle_arch
     celix_properties_t* bundleStateProperties = NULL;
     bundleStateProperties = celix_properties_load(archive->savedBundleStatePropertiesPath);
     if (bundleStateProperties == NULL) {
+        celix_framework_logTssErrors(archive->fw->logger, CELIX_LOG_LEVEL_DEBUG);
```

Comment:
Should this not be an CELIX_LOG_LEVEL_ERROR?

Reply:
Indeed, it should be an error.